### PR TITLE
feat(namespace): add azure-pg-admin-user label

### PR DESCRIPTION
**Pull request by douglasduteil**

### CODE REVIEW

**Comment by revolunet** [comment]
The label should be `azure-pg-admin-user=sample-next-app`

**Comment by douglasduteil** [comment]
K. I'm updating it...

**Comment by douglasduteil** [comment]
🆙 @revolunet

File: src/components/namespace/__snapshots__/index.test.ts.snap [hunk 1]
```diff
@@ -31,6 +32,7 @@ Object {
     },
     "labels": Object {
       "application": "sample",
+      "kubed.appscode.com/sync": "sample",
```
Comment (revolunet) [inline src/components/namespace/__snapshots__/index.test.ts.snap:35]: strange

Comment (douglasduteil) [reply src/components/namespace/__snapshots__/index.test.ts.snap:35]: missing file in commit 😊

**Review by revolunet — APPROVED** [review]

**Pull request merged**